### ISSUE TITLE
Performance benchmark question

### ISSUE THREAD

**Issue by tomsfernandez** [ssilverman/snowy-json#8]
Hi! Awesome work! By any chance de you have performance benchmarks comparing to other json schema validators in the jvm ecosystem?

**Comment by ssilverman** [comment]
Thanks! I’ve never done any benchmarking comparisons. In fact, while I do think this runs with reasonable performance, I’ve not spent much time optimizing it because my main goal is correctness.

Can you describe a little bit more about your needs?

**Comment by ssilverman** [comment]
[That’s weird. I don’t know how my comment disappeared. Let me try again.]

Thanks! I haven’t done any performance comparisons, but the speed I think is reasonable. My goal for the library is correctness, so I haven’t spent much time optimizing.

As far as I know, the Snow validator is one of a very few Java implementations that implement Draft 2019-09, so it might be hard to do comparisons for the latest features.

Can you tell me more about your needs?
